File: Medium/Python/2576. Find the Maximum Number of Marked Indices.py
class Solution(object):
    def maxNumOfMarkedIndices(self, nums):
        i = 0
        j = (len(nums) + 1) / 2
        nums.sort()

        while j < len(nums):
            if 2 * nums[i] <= nums[j]:
                i += 1
            j += 1
        
        return 2 * i

File: Medium/Java/2576. Find the Maximum Number of Marked Indices.java
class Solution {
    public int maxNumOfMarkedIndices(int[] nums) {
        int i = 0;
        int j = (nums.length + 1) / 2;
        Arrays.sort(nums);

        while(j < nums.length){
            if(2 * nums[i] <= nums[j])
                i++;
            j++;
        }

        return 2 * i;
    }
}
